File: 프로그래머스/Level1/로또의 최고 순위와 최저 순위.py
# https://programmers.co.kr/learn/courses/30/lessons/77484
def solution(lottos, win_nums):
    erase_num =  lottos.count(0)
    match_num = len(set(lottos)&set(win_nums))
    best = 7-erase_num-match_num
    if best > 6:
        best = 6
    worst = 7-match_num
    if worst > 6:
        worst = 6
    
    answer = [best,worst]
    return answer

File: 프로그래머스/Level1/로또의 최고 순위와 최저 순위.js
// https://programmers.co.kr/learn/courses/30/lessons/77484?language=javascript
function solution(lottos, win_nums) {
  let answer = [];
  let count = 0
  let zero_count = lottos.map( num => {
      if(num==0){
          count +=1
      }
  })
  let correct = lottos.filter(lotto => win_nums.includes(lotto));
  answer.push(7-correct.length-count);
  answer.push(7-correct.length);
  answer = answer.map(x => {
      if(x==7){
          return 6
      }else{
          return x
      }
  })
  return answer;
}